Purchases/Public/RCPurchases.m

```diff
@@ -587,7 +599,10 @@ - (void)        purchaseProduct:(SKProduct *)product
 
 - (void)syncPurchasesWithCompletionBlock:(nullable RCReceivePurchaserInfoBlock)completion {
     [self syncPurchasesWithReceiptRefreshPolicy:RCReceiptRefreshPolicyNever
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
                                       isRestore:self.allowSharingAppStoreAccount
+#pragma GCC diagnostic pop
```

Comment:
this prevents build warnings about using deprecated methods

Reply:
per the spec, we still want this parameter, because it might help us fix cases if something goes wrong backend-side.